packages/fetchai/connections/p2p_noise/connection.py

```diff
@@ -331,8 +331,9 @@ async def start(self) -> None:
             os.remove(in_path)
         if os.path.exists(out_path):
             os.remove(out_path)
-        os.mkfifo(in_path)
-        os.mkfifo(out_path)
+        # Ignore type-hinting check for Windows
+        os.mkfifo(in_path)  # type: ignore
+        os.mkfifo(out_path)  # type: ignore
```

Comment:
this needs to be fixed at some point. `os.mkfifo` is not available on Windows

Reply:
#1204